ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -308,6 +308,17 @@ FAN_SS_VSC_EFC_FDPM:
   - EFC
   - FDPM
 
+FAN_SS_VSC_WDPC_STM_FDPSM4X:
+  description: "Fan with start/stop status, speed control with fire, EC fan and filter status monitoring for purification"
```

Comment:
where is the fire data? either modify the point list or modify the description

Reply:
Modified the description